index.d.ts

```diff
@@ -112,6 +112,26 @@ export interface Options {
 	```
 	*/
 	readonly preserveLeadingUnderscore?: boolean;
+
+	/**
+	If your string ends with a dash, it will be preserved in the slugified string.
+
+	Sometimes trailing dashes are intentional, for example, when using slugify on an input field used for setting a slug.
```

Comment:
I think this should be clearer that the main use-case for this is to allow validation while not preventing the user to write a slug.

Reply:
Not sure how exactly to formulate it, could you suggest an edit?

Reply:
I think you should give it a try first.

Reply:
How about now? https://github.com/sindresorhus/slugify/pull/57/commits/2142006642c80006f65fd8b3248b596ce0770aee